packages/jest-preset-mc-app/package.json

```diff
@@ -39,14 +39,14 @@
     "jest-transform-graphql": "2.1.0",
     "jest-watch-master": "1.0.0",
     "jest-watch-typeahead": "0.3.1",
-    "make-plural": "4.3.0",
+    "make-plural": "5.0.0",
```

Comment:
They only extracted their CLI into a package we don't use. So we're not affected by this.